crates/cli/tests/test-projects/false-interpreter/main.roc

```diff
@@ -11,132 +11,124 @@ import Variable exposing [Variable]
 # It has some extra constraints:
 # 1) The input files are considered too large to just read in at once. Instead it is read via buffer or line.
 # 2) The output is also considered too large to generate in memory. It must be printed as we go via buffer or line.
+
+# All of the notes below this line are hopefully outdated with purity inference.
```

Comment:
nitpick: maybe prefix this with a TODO so we remember to remove the outdated notes if this successfully fixes the issue they point out?

Reply:
Good call. Removed. I think it is all resolved.

Reply:
Also, false feel surprisingly snappy for what it is.